apps/country/filters.py

```diff
@@ -56,6 +58,14 @@ class Meta:
             'id': ['iexact'],
         }
 
+    def filter_by_events(self, qs, name, value):
+        countries = qs.filter(events__in=value)
+        return countries.distinct()
+
+    def filter_by_crisis(self, qs, name, value):
+        countries = qs.filter(crises__in=value)
+        return countries.distinct()
```

Comment:
Let's do something like this, current one might create big join
```suggestion
    def filter_by_events(self, qs, name, value):
        if not value:
            return qs
        return qs.filter(
            id__in=Country.objects.filter(events__in=value).values('id')
        )

    def filter_by_crisis(self, qs, name, value):
        if not value:
            return qs
        return qs.filter(
            id__in=Country.objects.filter(crises__in=value).values('id')
        )
```